src/structurizr/model/element.py

```diff
@@ -83,12 +83,10 @@ def __repr__(self):
         return f"{type(self).__name__}(id={self.id}, name={self.name})"
 
     @property
+    @abstractmethod
     def child_elements(self) -> Iterable["Element"]:
-        """Return the elements that are children of this one.
-
-        Subclasses should override this to provide their specific children.
-        """
-        return []
+        """Return the elements that are children of this one."""
+        ...  # pragma: no cover
```

Comment:
I think `pass` is more clear than `...`.

Reply:
OK